index.js

```diff
@@ -1,5 +1,7 @@
 const Discordie = require('discordie')
 const UserVoice = require('uservoice-nodejs')
+const Dash = require('rethinkdbdash')
```

Comment:
This needs to be centralized at some point, but it's beyond the scope of this PR